azurerm/internal/services/datafactory/registration.go

```diff
@@ -43,6 +43,7 @@ func (r Registration) SupportedResources() map[string]*pluginsdk.Resource {
 		"azurerm_data_factory_integration_runtime_azure":             resourceDataFactoryIntegrationRuntimeAzure(),
 		"azurerm_data_factory_integration_runtime_azure_ssis":        resourceDataFactoryIntegrationRuntimeAzureSsis(),
 		"azurerm_data_factory_integration_runtime_self_hosted":       resourceDataFactoryIntegrationRuntimeSelfHosted(),
+		"azurerm_data_factory_linked_service":                        resourceDataFactoryLinkedService(),
```

Comment:
as requested could we make this
```suggestion
		"azurerm_data_factory_linked_custom_service":                        resourceDataFactoryLinkedService(),
```